sechub-pds-solutions/owasp-zap/docker/Owasp-Zap-Debian.dockerfile

```
@@ -4,62 +4,37 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
+LABEL org.opencontainers.image.source="https://github.com/mercedes-benz/sechub"
+LABEL org.opencontainers.image.title="SecHub GoSec+PDS Image"
+LABEL org.opencontainers.image.description="A container which combines GoSec with the SecHub Product Delegation Server (PDS)"
 LABEL maintainer="SecHub FOSS Team"
 
 # Build args
 ARG OWASPZAP_VERSION="2.12.0"
-ARG OWASPZAP_CHECKSUM="7eaf340d9fcc42576c7a5572249fe0bcad6e7acd68098a7ca110e64beab46207"
-ARG PDS_VERSION="0.33.0"
-
-ARG JAVA_VERSION="11"
-ARG PDS_FOLDER="/pds"
-ARG SCRIPT_FOLDER="/scripts"
-ARG USER="zap"
-ARG WORKSPACE="/workspace"
-
-# Environment vars
-ENV DOWNLOAD_FOLDER="/downloads"
-ENV MOCK_FOLDER="${SCRIPT_FOLDER}/mocks"
-ENV PDS_VERSION="${PDS_VERSION}"
-ENV SHARED_VOLUMES="/shared_volumes"
-ENV SHARED_VOLUME_UPLOAD_DIR="${SHARED_VOLUMES}/uploads"
-ENV TOOL_FOLDER="/tools"
-
-# non-root user
-# using fixed group and user ids
-# zap needs a home directory for the plugins
-RUN groupadd --gid 2323 "$USER" \
-     && useradd --uid 2323 --no-log-init --create-home --gid "$USER" "$USER"
+ARG OWASPZAP_SHA256SUM="7eaf340d9fcc42576c7a5572249fe0bcad6e7acd68098a7ca110e64beab46207"
 
 # Create folders & change owner of folders
-RUN mkdir --parents "$PDS_FOLDER" "${SCRIPT_FOLDER}" "$TOOL_FOLDER" "$WORKSPACE" "$DOWNLOAD_FOLDER" "MOCK_FOLDER" "$SHARED_VOLUME_UPLOAD_DIR" "/home/$USER/.ZAP/plugin"
+RUN mkdir --parents "/home/$USER/.ZAP/plugin"
+
+USER root
 
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get update && \
     apt-get upgrade --assume-yes && \
-    apt-get install --assume-yes wget openjdk-${JAVA_VERSION}-jre firefox-esr && \
+    apt-get install --assume-yes wget openjdk-11-jre firefox-esr && \
```

The Java JVM is already installed in the PDS base image.

```suggestion
    apt-get install --assume-yes firefox-esr && \
```

---

Regarding `wget`:
This package is mandatory here. It could happen that sombody removes it from the pds base image. The the build would fail.

On the other hand: Nothing happens when `wget` is already installed. (only an informational message from apt-get)

Because of this, I would keep it in the list here.

---

The one with Java8 is surprising.

---

About `wget`, you are correct, that is the beauty of package managers, they track what is already installed.

In my opinion this issue is resolved.